test/sagemaker_tests/pytorch/training/resources/dgl-gcn/gcn.py

```diff
@@ -1,192 +1,99 @@
-"""GCN using DGL nn package
-
-References:
-- Semi-Supervised Classification with Graph Convolutional Networks
-- Paper: https://arxiv.org/abs/1609.02907
-- Code: https://github.com/tkipf/gcn
-"""
-import argparse
-import os
-from dgl.data import CoraGraphDataset, CiteseerGraphDataset, PubmedGraphDataset
-import dgl
-import torch.nn.functional as F
-import numpy as np
-import time
 import torch
```

Comment:
Does the changes to this file corresponds to https://github.com/dmlc/dgl/blob/0.8.x/examples/pytorch/gcn/gcn.py ?

Reply:
it's corresponding to https://github.com/dmlc/dgl/blob/master/examples/pytorch/gcn/train.py